requirements.txt

```
@@ -1,2 +1,3 @@
 sphinx
 sphinx_rtd_theme
+sphinxemoji.sphinxemoji
```

travis failed to run pip install -r requirements

isn't it 
```suggestion
sphinxemoji
```
?